src/UI/Component/Modal/Lightbox.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace ILIAS\UI\Component\Modal;
+
+use ILIAS\UI\Component\Button;
+
+/**
+ * Interface Lightbox
+ *
+ * @package ILIAS\UI\Component\Modal
+ */
+interface Lightbox extends Modal
+{
+
+	/**
+	 * Get a modal like this with the given lightbox pages.
+	 *
+	 * @param LightboxPage[] $pages
+	 *
+	 * @return Lightbox
+	 */
+	public function withPages(array $pages);
```

Comment:
I don't think we need that. Why would I want to change the pages when all a lightbox is is a collection of pages?